src/components/Calculator.js

```diff
@@ -1,26 +1,175 @@
-import { Component } from 'react';
-import LeftView from './LeftView';
-import RightView from './RightView';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+import operate from '../logic/operate';
```

Comment:
### Optional
- [ ] There is no need to import the operate file here because it is already imported in the calculate.js file